src/main/java/Duke.java

```diff
@@ -1,10 +1,88 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
+        String line;
+        Task[] list = new Task[100];
```

Comment:
you might want to use taskList instead of list when there are more lists in the future

Reply:
Thanks, will consider it!